schema/articles.js

```diff
@@ -38,7 +38,7 @@ export default {
         positiveFeedbackCount: { type: 'long' },
         negativeFeedbackCount: { type: 'long' },
 
-        // One reply can have multipe replyconnections.
+        // One reply can have multiple replyconnections.
```

Comment:
Thanks for updating the typo!
I found that the text is a bit outdated as well.

```suggestion
        // One reply can have multiple articlereplies.
```